### PR TITLE
Add CLI Feature: Hot commands

### PR DESCRIPTION
**Pull request by sakshamb2113**
This PR is for https://github.com/onflow/flip-fest/issues/14

## Description

- Customised the help screen to display commands classified into 2 types.
- Added 2 new "Hot Commands"- deploy and run.
- Used Annotation field in the cobra.Command class to classify the commands.
- Now help screen looks as shown below
Screenshot from 2021-10-30 17-43-00

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: internal/quick/deploy.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/internal/project"
+)
+// add deploy command
+var DeployCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "deploy",
+		Short:   "Deploy contracts",
```
Comment (sideninja) [inline internal/quick/deploy.go:31]: Deploy all project contracts

File: internal/quick/run.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// add run command
+var RunCommand = &cobra.Command{
+	Use:     "run",
+	Short:   "Start emulator and deploy contracts",
+	Example: "flow run",
+	Annotations: map[string]string{
+		"HotCommand": "true",
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		project.DeployCommand.Cmd.Run(cmd, args)
```
Comment (sideninja) [inline internal/quick/run.go:35]: Wouldn't this two commands be swapped in order?

**Review by sideninja — CHANGES_REQUESTED** [review]
Great effort so far, one thing that is problematic and needs changing is the fact that if you now use help with any of the commands you get a "hot commands" section, so for example, if I try to run `flow accounts --help` I will get first an empty "hot commands" section, but the way we want this to work is to only show that on the top "flow" command help.

**Comment by sakshamb2113** [comment]
I have made the changes and updated the code. Could you please have a look at it now ?

**Review by sideninja — APPROVED** [review]
Great improvement, LGTM

**Pull request merged**